src/utils/Processing.ts

```diff
@@ -240,6 +241,29 @@ export function fillTank(rawFSHes: RawFSH[], config: Configuration): FSHTank {
   return new FSHTank(docs, config);
 }
 
+export function checkNullValuesOnArray(resource: any, parentName = ''): void {
+  const resourceName = parentName ? parentName : resource.id ?? resource.name;
+  for (const property in resource) {
+    // We'll want to exclude properties beginning with an "_" as those are arrays containing extensions
+    // corresponding to a primitive array, per the FHIR spec
+    if (property.startsWith('_')) continue;
```

Comment:
Unfortunately, I think it might be a little more complicated than this.  It is true that if a property starts with `_` and is an array, it is OK for it to have null values. But... each element of the array that is not null also might contain a sub-property that is an array and maybe should not have nulls.

For example, in this case:
```json5
{
  "_foo": [
    null, // this is ok
    {
      "extension": [
        null, // this is NOT ok
        { "url": "http://foo.org", "valueBoolean": true }
      ]
    }
  ]
} 
```